### ISSUE TITLE
Requesting for resources using kubectl throws an error

### ISSUE THREAD

**Issue by rudimk** [rancher/k3os#47]
I'm running the latest release of k3OS on VirtualBox 6.0.6 r130049. Running `kubectl get nodes` throws the following error:

```The connection to the server localhost:6443 was refused - did you specify the right host or port?```

I tried other resources - I get the same result. At first I thought there may be an issue with the VM's networking - but on further reflection, I no longer think so. Probably a configuration issue of some sort.

Attaching a screenshot. 

![image](https://user-images.githubusercontent.com/288007/56797336-20d51600-6832-11e9-9990-33ea335545c9.png)

**Comment by kewinbrand** [comment]
Any update on this? I'm facing exactly the same problem

**Comment by rudimk** [comment]
I haven't played with it since. I've got some new bare metal servers coming in next week, and I'm thinking I'll try running this on KVM, see if the issue still persists.

**Comment by matjohn2** [comment]
+1 same issue. Both live booted and once installed. Occurs on KVM-based and bare metal.

**Comment by matjohn2** [comment]
OK, found it.

**TL;DR** There's probably something up with your DNS settings, which you're probably getting via DHCP. Test this by changing `/etc/resolv.conf` to just read `nameserver 8.8.8.8` and removing any `search` statements and see if everything starts working within a couple of mins (ie `kubectl get nodes` works).

**Comment by dweomer** [comment]
Fixed via #170
